fastestimator/network.py

```diff
@@ -40,8 +40,9 @@ def _verify_inputs(self):
             else:
                 assert isinstance(op, TensorOp), "unsupported op format, must provide TensorOp in Network"
 
-    def load_epoch(self, mode: str, epoch: int):
-        self.epoch_ops = get_current_ops(self.ops, mode, epoch)
+    def load_epoch(self, mode: str, epoch: int) -> Set[TensorOp]:
```

Comment:
are epoch ops really a set? seems like conceptually this would be a list

Reply:
nice catch!